ADA/git_databricks.qmd

```diff
@@ -52,29 +46,26 @@ The connection between git and Databricks is established using a secure access t
 
 Additionally, when you commit and push notebooks through the Databricks interface, it will automatically clear any output cells from your notebook, meaning that you cannot accidentally include any unpublished data in there.
 
-\
-
 ------------------------------------------------------------------------
 
-# Setting up a connection to Azure DevOps
-
-------------------------------------------------------------------------
+## Setting up a connection to Azure DevOps
 
-## Prerequisites
+### Prerequisites
 
+---
 
 - An Azure DevOps account and access to the repo you need to connect to
 - A Databricks account and access to your notebooks
 
-\
-
 ------------------------------------------------------------------------
 
-## Getting set up
+### Getting set up
 
+---
 
-### Access Tokens
+#### Access Tokens
 
+---
 
 Access tokens are long strings of numbers and letters that act like a password between two services. They identify the user and their permissions from one service to another. 
 
```

Comment:
Line 86 (in the bullet points) there's a lower case git used

Reply:
Yeah this one is ok too for me locally - will commit and push again